### PR TITLE
Update all the openSUSE images

### PR DESCRIPTION
**Pull request by flavio**
Introduce openSUSE Leap 42.1.
Update openSUSE 13.2 and Tumbleweed images.

Signed-off-by: Flavio Castelli <fcastelli@suse.com>

### CODE REVIEW

File: library/opensuse [hunk 1]
```diff
@@ -1,9 +1,14 @@
 # maintainer: Flavio Castelli <fcastelli@suse.com> (@flavio)
 
 # openSUSE 13.2
-13.2: git://github.com/openSUSE/docker-containers-build@f3c7e8aa49da4c10b3352d56c28d3cea6f3afefa docker
-harlequin: git://github.com/openSUSE/docker-containers-build@f3c7e8aa49da4c10b3352d56c28d3cea6f3afefa docker
-latest: git://github.com/openSUSE/docker-containers-build@f3c7e8aa49da4c10b3352d56c28d3cea6f3afefa docker
+13.2: git://github.com/openSUSE/docker-containers-build@d9e75cd826dffd35984b8c7c9322494810b4e29c docker
+harlequin: git://github.com/openSUSE/docker-containers-build@d9e75cd826dffd35984b8c7c9322494810b4e29c docker
+
+# openSUSE 13.2
```
Comment (tianon) [inline library/opensuse]: Lol 13.2 again? :smile:

File: library/opensuse [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
+42.2: git://github.com/openSUSE/docker-containers-build@da3cf11b9e96b61ffbe92de846e77075bc11e326 docker
```
Comment (tianon) [inline library/opensuse]: Was this supposed to be `42.1`, or is `42.2` correct?

**Comment by flavio** [comment]
Crap, I should stop doing that stuff late at night :/

Thanks @tianon for the heads up!

**Comment by yosifkit** [comment]
LGTM, Build test of #1172; 4e88bcdce343532c4bc366e3fa5bbeb605b5b200 (`opensuse`):

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/4e88bcdce343532c4bc366e3fa5bbeb605b5b200/library/opensuse"
$ bashbrew build "$url"
Fetching opensuse (git://github.com/openSUSE/docker-containers-build) ...
Processing opensuse:13.2 ...
Processing opensuse:harlequin ...
Processing opensuse:42.1 ...
Processing opensuse:leap ...
Processing opensuse:latest ...
Processing opensuse:tumbleweed ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing opensuse:13.2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...awk: fatal: cannot open file `/etc/shadow' for reading (No such file or directory)
warning: garbage password detected for 'uucp': 'x'
warning: garbage password detected for 'games': 'x'
warning: garbage password detected for 'bin': 'x'
warning: garbage password detected for 'mail': 'x'
warning: garbage password detected for 'ftp': 'x'
warning: garbage password detected for 'daemon': 'x'
warning: garbage password detected for 'root': 'COI22ZqusHqVo'
warning: garbage password detected for 'wwwrun': 'x'
warning: garbage password detected for 'man': 'x'
warning: garbage password detected for 'nobody': 'x'
warning: garbage password detected for 'news': 'x'
warning: garbage password detected for 'lp': 'x'
passed
	'override-cmd' [4/4]...passed
testing opensuse:42.1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': '087cPKTh41EkU'
passed
	'override-cmd' [4/4]...passed
testing opensuse:tumbleweed
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': '9vtGOrG2r11R6'
passed
	'override-cmd' [4/4]...passed
```

**Comment by flavio** [comment]
Hi, the images are not yet reachable, is everything fine? I'm asking because in the past there was a really short delay between the merge of the PR and the images being available for pull.

**Comment by yosifkit** [comment]
Sorry, the build server was sad when it ran out of space for docker images last night; sometimes we forget to clean up the untagged images.  It it pushed now :smile:.

**Pull request merged**